packages/google-types/src/models.ts

```diff
@@ -1,3 +1,3 @@
-import { oauth2_v2 } from 'googleapis';
+import { oauth2_v2 } from 'googleapis/build/src/apis/oauth2/v2';
```

Comment:
does this have an impact on build time? even with skipLibCheck?

Reply:
yes, `skipLibCheck` doesn't validate types inside the libs, but it parses all the types, so this line speedups building for ~second